packages/gatsby-transformer-sqip/src/generate-sqip.js

```diff
@@ -0,0 +1,80 @@
+const crypto = require(`crypto`)
+const { resolve, parse } = require(`path`)
+
+const Debug = require(`debug`)
+const { exists, readFile, writeFile } = require(`fs-extra`)
+const svgToMiniDataURI = require(`mini-svg-data-uri`)
+const PQueue = require(`p-queue`)
+const sqip = require(`sqip`)
+
+const queue = new PQueue({ concurrency: 1 })
```

Comment:
Why not set the concurrency higher? Often people set it at the number of CPUs - 1

Reply:
SQIP (Primitive) is using all CPU cores by default and this is currently not configurable via SQIP. Plus, the whole script is using multiple cores to generate exact previews. We have to keep it like this